packages/core/src/Stores/traders-hub-store.js

```diff
@@ -622,7 +623,12 @@ export default class TradersHubStore extends BaseStore {
         );
     }
     getAvailableCTraderAccounts() {
-        if (this.CFDs_restricted_countries || this.financial_restricted_countries) {
+        if (
+            this.CFDs_restricted_countries ||
+            this.financial_restricted_countries ||
+            (this.root_store.client.is_logged_in &&
+                JSON.parse(JSON.stringify(this.root_store.client.landing_companies.ctrader.all.standard)) === 'none')
```

Comment:
Is the JSON.parse and followed by JSON.stringify even necessary?
```suggestion
                this.root_store.client.landing_companies.ctrader.all.standard === 'none'
```